values.yaml

```diff
@@ -14,7 +14,12 @@ fullnameOverride: ""
 
 image:
   repository: softonic/homing-pigeon
-  tag: 0.1.0
+  tag: latest
```

Comment:
Please specify sensible default value

Reply:
There aren't a current release for that support this chart, so I can't specify it yet.

Reply:
I believe 0.1.0 also was not available :stuck_out_tongue: